src/Widgets/DiskBar.vala

```diff
@@ -148,7 +123,7 @@ public class Installer.DiskBar: Gtk.Grid {
             event_box.add (widget);
             event_box.add_events (Gdk.EventMask.BUTTON_PRESS_MASK);
             event_box.button_press_event.connect (() => {
-                menu.popup ();
+                menu.popup();
```

Comment:
Weird merge artifact? Space before paren is proper code style.

Reply:
It could be that it was fixed upstream, but not in our fork.